src/main/java/at/gammastrahlung/monopoly_server/game/Game.java

```diff
@@ -245,4 +249,66 @@ public enum GameState {
          */
         ENDED
     }
+
+    // Starts a new auction for a specific property
+    public void startAuction(Property auctionProperty) {
+        currentAuction = new AuctionSystem(auctionProperty, new AuctionSystem.AuctionEventListener() {
+            @Override
+            public void onBidUpdated(Player player, int newBid) {
+                // Implement what should happen when a bid is updated
```

Comment:
You commented what you have to do.
Also don't send something to the players in the Game class